internal/backend/storage/fs/store.go

```diff
@@ -151,7 +151,7 @@ func (s *Store) Exists(ctx context.Context, name string) bool {
 	}
 	path := filepath.Join(s.path, filepath.Clean(name))
 	found := fsutil.IsFile(path)
-	debug.Log("Checking if %s exists at %s: %t", name, path, found)
+	debug.Log("Checking if '%s' exists at %s: %t", name, path, found)
```

Comment:
nit: I think you could use `%q` instead of `'%s'`.

Reply:
Nice, I never use `%q`, I should.